src/azure-cli/azure/cli/command_modules/storage/operations/azcopy.py

```diff
@@ -65,6 +65,18 @@ def storage_remove(cmd, client, service, target, recursive=None, exclude_pattern
     azcopy.remove(_add_url_sas(target, sas_token), flags=flags)
 
 
+# pylint: disable=unused-argument
+def storage_fs_directory_copy(cmd, source, destination, recursive=None, **kwargs):
+    azcopy = AzCopy()
+    if kwargs.get('token_credential'):
+        azcopy = _azcopy_login_client(cmd)
+
+    flags = []
+    if recursive is not None:
```

Comment:
when it is true, you could add the flag.

Reply:
done